src/modules/endpoint/__tests__/EndpointBuilder.js

```diff
@@ -0,0 +1,52 @@
+import EndpointBuilder from '../EndpointBuilder'
```

Comment:
-> EndpointBuilder.spec.js

Reply:
you should also rename the location endpoint to locations and pages to categories

Reply:
The endpoints are named like in our backend. I'll rename language to languages though.